openshift-ci/Dockerfile.bundle.ci

```diff
@@ -11,4 +11,5 @@ LABEL operators.operatorframework.io.bundle.package.v1=performance-addon-operato
 LABEL operators.operatorframework.io.bundle.channels.v1=4.6.0
 LABEL operators.operatorframework.io.bundle.channel.default.v1=4.6.0
 
-COPY ${OLM_SOURCE}/performance-addon-operator/${CHANNEL}.0/ /
+COPY deploy/metadata/performance-addon-operator/${CHANNEL}.${ZVERSION}/* /metadata/
+COPY deploy/olm-catalog/performance-addon-operator/${CHANNEL}.${ZVERSION}/* /manifests/
```

Comment:
could you please elaborate why this is needed? our CI is already producing manifests bundle, so is not clear to me why this needs to be fixed:
https://quay.io/repository/openshift-kni/performance-addon-operator-bundle

Reply:
@fromanirh I need it to build the operator under the CI, and looks like when it build the operator it uses the src image as base and the src image does not have the `build/_output` directory at all because it created once we are running `make generate`.
Anyway, I think it not so big deal, because it does not affect our d/s at all and we also did not use this image during the u/s testing.

Reply:
Problem is just we can fetch the wrong manifests (from a wrong place)

Reply:
ok, we also had further discussion online. This can indeed work. The flow is a but too messy in general, but we will clean up later.